monkey/monkey_island/cc/consts.py

```diff
@@ -3,3 +3,4 @@
 __author__ = 'itay.mizeretz'
 
 MONKEY_ISLAND_ABS_PATH = os.path.join(os.getcwd(), 'monkey_island')
+DEFAULT_MONKEY_TTL_EXPIRY_DURATION_IN_SECONDS = 60 * 5
```

Comment:
7 words in a variable name seem too long. I think DEFAULT_MONKEY_TTL is enough, but it doesn't really matter as it's not used that often.

Reply:
If you don't mind leaving it as is then I prefer to leave it. I always prefer explicitness 📖